spec/mempool/mempool.md

```diff
@@ -0,0 +1,152 @@
+# Mempool
+
+In this document, we define the notion of **mempool** and characterize its role in the **CometBFT** protocol.
+First, we provide an overview of what is a mempool, and relates it to other blockchains.
+Then, the interactions with the consensus and client application are detailed.
+A formalization of the mempool follows.
+This formalization is readable in Quint [here](https://github.com/cometbft/cometbft/blob/main/spec/mempool/quint).
+
+## Overview
+
+The mempool acts as an entry point to consensus.
+It permits to disseminate transactions from one node to another, for their eventual inclusion in the blockchain.
+To this end, the mempool maintains a replicated set, or _pool_, of transactions.
+Transactions in the mempool are consumed by consensus to create the next proposed block.
+The mempool is refreshed once a new block in the blockchain  is decided.
+
+A transaction can be received from a local client, or a remote disseminating process.
+Each transaction is subject to a test by the client application.
```

Comment:
The use of the term client is confusing here. Not sure what is meant with "local client". I guess "client application" should be "ABCI application"?

Reply:
I added such a precision.